src/RawModelExportItem.tsx

```diff
@@ -0,0 +1,91 @@
+import Stack from '@mui/material/Stack'
+import Typography from '@mui/material/Typography'
+import Box from '@mui/system/Box'
+import Button from '@mui/material/Button'
+import { useState } from 'react'
+import DialogTitle from '@mui/material/DialogTitle'
+import DialogContent from '@mui/material/DialogContent'
+import Dialog from '@mui/material/Dialog'
+import { Divider } from '@mui/material'
+import DirectoryTreeView from './DirectoryTreeView'
+import { ModelUploadType } from '../types/interfaces'
+import DisabledElementTooltip from './common/DisabledElementTooltip'
+
+function RawModelExportItem({
+  deploymentUuid,
+  version,
+  uploadType,
+}: {
+  deploymentUuid: string
+  version: string
+  uploadType: string
+}) {
+  const [displayTree, setDisplayTree] = useState(false)
+
+  let primaryButtonDisabledReason: string | undefined = 'No files to download'
+  if (uploadType === ModelUploadType.Zip || uploadType === undefined) {
+    primaryButtonDisabledReason = undefined
+  }
+
+  const hideTree = () => {
+    setDisplayTree(false)
+  }
+
+  return (
+    <Box key={version}>
```

Comment:
We shouldn't be providing a key here, it does nothing. I've double checked the consumers of this component to see if we should be providing keys there instead, but we do not. Therefore this `Box` can be swapped out for a fragment `<>`.